src/lavinmq/http/controller/logs.cr

```diff
@@ -0,0 +1,42 @@
+require "../controller"
+require "../../sortable_json"
+require "../../config"
+require "../http_server"
+require "../../in_memory_backend"
+
+module LavinMQ
+  module HTTP
+    class LogsController < Controller
+      private def register_routes
+        get "/api/livelog" do |context, _params|
+          channel = ::Log::InMemoryBackend.instance.add_channel
+          context.response.content_type = "text/event-stream"
+          context.response.upgrade do |io|
+            if last_event_id = context.request.headers["Last-Event-Id"]?
+              last_ts = last_event_id.to_i64
+              ::Log::InMemoryBackend.instance.entries.each.skip_while { |e| e.timestamp.to_unix_ms <= last_ts }.each do |entry|
+                print_entry(entry, io)
+              end
+            else
+              ::Log::InMemoryBackend.instance.entries.each do |entry|
+                print_entry(entry, io)
+              end
+            end
+            while entry = channel.receive
+              print_entry(entry, io)
+            end
+          ensure
+            ::Log::InMemoryBackend.instance.remove_channel(channel)
+          end
+          context
+        end
+      end
+
+      private def print_entry(entry, io)
+        io.print("id: #{entry.timestamp.to_unix_ms}\n")
+        io.print("data: #{entry.timestamp.to_utc} #{entry.severity} #{entry.source} #{entry.message}\n\n")
```

Comment:
> no, i was thinking of just that. how can we make it bigger?

It depends on setup I guess, but in some cases it could be possible to download the actual log files? If we know where the log files are stored and have access to them. So, maybe not when using stdout/journal for logging, but if you've specified a logfile in config. 

But I agree, we should just implement a download for this solution for now.

Reply:
> should make sure that `entry.message` can't include '\n', or prefix each line in message with `data: `. lines that doesn't start with `data:` will be ignored by sse.

Yes, I'll fix. I think it's best to prefix with `data: ` if `\n` exists in message